packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp

```diff
@@ -141,6 +141,9 @@ setParameters (const Teuchos::ParameterList& List)
     if (relaxType == "Jacobi") {
       PrecType_ = Ifpack2::Details::JACOBI;
     }
+    else if (relaxType == "MTJacobi") {
+      PrecType_ = Ifpack2::Details::MTJACOBI;
+    }
```

Comment:
"MT Jacobi" (with a space, which I should put in) would batch @mndevec's approach: "MT Gauss Seidel". I don't object to changing convention, however. "Multithreaded Jacobi"?